test/feature/mac/agent_config.json

```diff
@@ -36,43 +38,35 @@
             ],
             "metrics_collection_interval": 1
           },
-          "mem": {
+        "mem": {
             "measurement": [
               "available_percent", 
               "used_percent"
             ],
             "metrics_collection_interval": 1
-          },
-          "diskio": {
+        },
+        "diskio": {
             "resources": [
-              "devfs"
+              "*"
             ],
             "measurement": [
               "iops_in_progress", 
               "io_time"
             ],
             "metrics_collection_interval": 1
-          },
-          "disk": {
+        },
+        "disk": {
             "resources": [
-              "devfs"
+              "*"
             ],
             "measurement": [
               "free",
               "used_percent"
             ],
-            "drop_device": true
-          },
-          "ethtool": {
```

Comment:
Ethtool is [not supported in MacOS](https://github.com/influxdata/telegraf/blob/6a3b27126a26dd43d2a287b4c9d8831dca6bf687/plugins/inputs/ethtool/ethtool_notlinux.go) and only supported in Linux since we are using [ethtool from telegraf](https://github.com/influxdata/telegraf/tree/6a3b27126a26dd43d2a287b4c9d8831dca6bf687/plugins/inputs/ethtool). Even though [CWA rules supports Darwin Rules](https://github.com/aws/amazon-cloudwatch-agent/blob/c85501042b088014ec40b636a8b6b2ccc9739738/translator/translate/metrics/metrics_collect/ethtool/ethtool.go#L59), it is not possible for collecting ethtool metrics in MacOS